packages/just-scripts-utils/src/transform.ts

```diff
@@ -2,30 +2,126 @@ import glob from 'glob';
 import path from 'path';
 import fse from 'fs-extra';
 import handlebars from 'handlebars';
+import { logger } from './logger';
 
-export function transform(srcPath: string, destPath: string, data?: any) {
-  const templateFiles = [...new Set([...glob.sync('**/*', { cwd: srcPath }), ...glob.sync('**/.*', { cwd: srcPath })])];
+export interface ITransformResult {
```

Comment:
I'm very open to suggestions for how the transform ought to handle errors and warnings. Current approach:
- Do informative warning/error logging (including the exception thrown where relevant)
- Return an object indicating:
  - whether an error occurred (defined as total failure to process anything from the template)
  - number of warnings (defined as failures to process individual file(s)/dir(s) from the template)
  - number of successfully processed files/dirs (might not be needed)

I don't love returning a big awkward object, but it was the best way I could think of to convey that the transform was partially successful (not just having binary success/error states).